include/cliffordsynthesis/Configuration.hpp

```diff
@@ -16,6 +16,7 @@ struct Configuration {
 
   /// General configuration for the synthesis algorithm
   std::size_t    initialTimestepLimit    = 0U;
+  std::size_t    minimalTimeSteps        = 0U;
```

Comment:
```suggestion
  std::size_t    minimalTimesteps        = 0U;
```
For consistency. Otherwise you would have to change the other occurrences to `minimal_time_steps`.
Another possibility: name this `minimalTimestepLimit` in accordance with the parameter right above it.

Reply:
Yeah that's a typo.